### ISSUE TITLE
Possible support for Meta MMS?

### ISSUE THREAD

**Issue by OkGoDoIt** [ggml-org/whisper.cpp#950]
I'm curious if anyone has had a chance to compare the model architecture of Meta's new [Massively Multilingual Speech (MMS)](https://github.com/facebookresearch/fairseq/tree/main/examples/mms) models to see if this system could use those models? They are claiming a massive reduction in word error rate, as well as support for over 1000 languages. It's unclear how performant MMS is likely to be but I'm sure someone here will look into it. I'd love to hear any thoughts or notes from anyone who is better equipped to understand how these two projects might align.

![image](https://github.com/ggerganov/whisper.cpp/assets/1185444/e048b0eb-c93b-40fb-aa14-f19834467062)

**Comment by G2G2G2G** [comment]
agreed, whisper is OK at clear English spoken to it.. but it's pretty bad at video subtitling. Any background noise I guess is what it is, makes it's timings way off is the biggest issue but also words start to be wrong.

hoping MMS is truly better, would have to test though

**Comment by benxh1995** [comment]
This will allow for very interesting novel uses of sound based interfaces, e.g. for edge case languages.

**Comment by noe** [comment]
Note that the [license of the model](https://github.com/facebookresearch/fairseq/tree/main/examples/mms#license) is [CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/), so it does not allow commercial use.

**Comment by kevin01881** [comment]
Our lord and savior @ggerganov will come to the rescue soon! 🙏

**Comment by Devashhag** [comment]
I mean just look at the numbers whisper is trained at 680 k hrs labelled data ,and look at meta only 45k hours labelled data train ,whisper will work well cause its trained significantly large while meta only 45 k hrs trained therefore their error rate is very low

**Comment by EliasVansteenkiste** [comment]
> I mean just look at the numbers whisper is trained at 680 k hrs labelled data ,and look at meta only 45k hours labelled data train ,whisper will work well cause its trained significantly large while meta only 45 k hrs trained therefore their error rate is very low

I think it's only benchmarking better at languages with fewer data.
So for larger languages, it's actually way worse:

See this table in the appendix of the paper: 
<img width="746" alt="Screenshot 2023-06-05 at 14 25 31" src="https://github.com/ggerganov/whisper.cpp/assets/2494383/e3cb3c63-3074-43ce-889b-4f7cf03566d9">

**Comment by guranu** [comment]
So uhh can metas MMS models be used in whisper.cpp?
